input/vulnerabilities/TowelRoot.json

```diff
@@ -33,5 +33,12 @@
     , "component" : "kernel"
     , "commit" : "e9c243a5a6de0be8e584c604d353412584b592f8"
     }
-  }
+  },
+  "Surface": ["local", "app"],
+  "Vector": ["memory-corruption"],
+  "Target": ["kernel"],
+  "Channel": ["app-execution"],
+  "Condition": ["unknown-source-install-allowed"],
```

Comment:
Is that really a precondition for this vulnerability? If you can run code in user mode you can get to root.

Reply:
Changed (and for several similar ones as well)